sound/soc/sof/topology.c

```diff
@@ -725,6 +725,9 @@ static const struct sof_topology_token comp_tokens[] = {
 	{SOF_TKN_COMP_FORMAT,
 		SND_SOC_TPLG_TUPLE_TYPE_STRING, get_token_comp_format,
 		offsetof(struct sof_ipc_comp_config, frame_fmt), 0},
+	{SOF_TKN_COMP_REF,
```

Comment:
A lot of this looks like it should be seperate feature/fix patches.

Reply:
well, we can make it a separate commit, but it's only needed for virtualisation (so far at least), so, as a separate patch motivation would be unclear.